lib/presentation/widgets/summary_rectangle.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/cupertino.dart';
+import 'package:trip_tally/presentation/utils/enums/context_extensions.dart';
+
+import '../theme/app_colors.dart';
+import '../theme/app_dimensions.dart';
+
+class SummaryRectangle extends StatelessWidget {
+  const SummaryRectangle({
+    super.key,
+    required this.spendMoney,
+    required this.budgetMoney,
+  });
+
+  final String spendMoney;
+  final String budgetMoney;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Container(
+          width: AppDimensions.d400,
```

Comment:
it's better to use padding than constant width values. This will be always 400 even if the screen size is 350 and it will break, unlike padding, it will always repel itself from the edge of the screen, so on a smaller screen the container will also be smaller